### DISCUSSION TITLE
Is YOLOv8 ready for production yet?

### DISCUSSION THREAD

**Discussion by haimat** [ultralytics/ultralytics#865] [General]
We use YOLOv5 in serveral projects and that systems works well and reliably so far. We are quite happy with that framework. Now as indicated by the performance charts on the new Ultralytics repo the performance of YOLOv8 should even be better. But I also see that some features are not (fully) implemented yet in YOLOv8 - which leads me to my question: Can we savely use that new framework in production at this time, or should we rather just experiment with it and stick to YOLOv5 in production for now?

**Comment by glenn-jocher** [comment]
@haimat hello! I'm glad to hear that YOLOv5 has been working well for you in production. YOLOv8 indeed offers performance improvements and is designed to be production-ready. However, as with any new framework, it's important to thoroughly test it within your specific use case before fully transitioning.

While some features may still be under development, the core functionalities for detection tasks are in place. If these core features align with your production needs, you can consider integrating YOLOv8 into your systems. However, if you rely on any features that are not yet fully implemented, it would be prudent to continue with YOLOv5 for those aspects of your production environment and experiment with YOLOv8 in parallel until it fully meets your requirements.

For detailed information on the current capabilities and updates of YOLOv8, you can refer to our documentation. 😊

**Comment by haimat** [comment]
Hi Glenn, thanks for your reply. My question is quite old, a year to be precise :)
In the meantime we have upgraded our YOLOv5-based software to YOLOv8 and have not any regrets since.

Thanks a lot for this greate framework!

**Reply by pderrenger** [reply]
@haimat hello! It's wonderful to hear that you've successfully upgraded to YOLOv8 and are enjoying the benefits of the new framework. Your positive feedback is greatly appreciated, and it's the community's support that drives the continuous improvement of our models. If you have any further questions or need assistance, don't hesitate to reach out. Happy detecting! 😊🚀

**Reply by glenn-jocher** [reply]
@haimat better late than never they say.
